test/semantic.test.js

```diff
@@ -42,3 +43,124 @@ describe('patch(version)', () => {
     expect(() => patch('not_version')).toThrowError(TypeError);
   });
 });
+
+describe('increment(version, major)', () => {
+  const testCases = [
+    // [ existing version, release type, identifier (`a`, `b`, `rc`, etc.), expected result ]
+
+    // Semantic Version numbers only.
+    ['0.0.0', 'major', undefined, '1.0.0'],
+    ['1.0.0', 'major', undefined, '2.0.0'],
+    ['1.0.1', 'major', undefined, '2.0.0'],
+    ['1.1.0', 'major', undefined, '2.0.0'],
+    ['0.0.0', 'minor', undefined, '0.1.0'],
+    ['1.0.0', 'minor', undefined, '1.1.0'],
+    ['1.0.1', 'minor', undefined, '1.1.0'],
+    ['1.1.0', 'minor', undefined, '1.2.0'],
+    ['0.0.0', 'patch', undefined, '0.0.1'],
+    ['1.0.0', 'patch', undefined, '1.0.1'],
+    ['1.0.1', 'patch', undefined, '1.0.2'],
+    ['1.1.0', 'patch', undefined, '1.1.1'],
+
+    // Extra release segments, as allowed by PEP440.
+    ['1.1.1.1', 'major', undefined, '2.0.0.0'],
+    ['1.1.1.1', 'minor', undefined, '1.2.0.0'],
+    ['1.1.1.1', 'patch', undefined, '1.1.2.0'],
+
+    // Without padding, as allowed by PEP440.
+    ['1', 'major', undefined, '2'],
```

Comment:
Should `inc` pad the string with extra zeros for the `minor` and `patch` release segments?

Reply:
I think that padding would be better implemented at its own function

Reply:
Okay! :+1: 

Based on your other comment about removing extra release segments, I can definitely see needing/wanting to create a function that coerces the PEP440 version string to a Semantic Version string where a Semantic Version string is specifically needed.